bindings/core/src/method/client.rs

```diff
@@ -125,6 +125,15 @@ pub enum ClientMethod {
     /// Returns the unhealthy nodes.
     #[cfg(not(target_family = "wasm"))]
     UnhealthyNodes,
+    /// Extension method which provides request methods for plugins.
+    #[serde(rename_all = "camelCase")]
+    CallPluginRoute {
+        base_plugin_path: String,
+        method: String,
+        method_path: String,
```

Comment:
I think this is a bit confusing given we also have method, maybe endpoint instead? Or something else

Reply:
but its also basePluginPath. base_plugin_endpoint as well?

Reply:
I don't have a strong opinion, I just don't like method_path :trollface:

Reply:
Are we changing it then?

Reply:
sure ill change method_path to endpoint